src/components/annual-report/report-data/SectionOne.js

```diff
@@ -77,64 +87,300 @@ const reportQueries = {
+                        </TableRowHead>
+                    </TableHead>
+                    <TableBody>
+                        {sectionData
+                            ? Object.values(
+                                  sectionData.section1.section1B
+                              ).map((dataset, key) => (
+                                  <TableRow key={key}>
+                                      <TableCell>
+                                          {dataset.dataset_name}
+                                      </TableCell>
+                                      <TableCell>
+                                          {dataset.threshold}%
+                                      </TableCell>
+                                      <TableCell>{dataset.score}%</TableCell>
+                                      <TableCell>
+                                          {dataset.divergentRegionsCount}
+                                      </TableCell>
+                                      <TableCell>
+                                          {dataset.divergentRegionsPercent}%
+                                      </TableCell>
+                                      <TableCell>
+                                          {dataset.orgUnitLevelsOrGroups.join(
+                                              ', '
+                                          )}
+                                      </TableCell>
+                                  </TableRow>
+                              ))
+                            : ''}
+                    </TableBody>
+                </Table>
+                <p>1c: Timeliness of facility reporting</p>
+                <p>
+                    The percentage of expected reports that have been entered
+                    and completed on time.
+                </p>
+                <Table>
+                    <TableHead>
+                        <TableRowHead>
+                            <TableCellHead>Indicator</TableCellHead>
+                            <TableCellHead>Quality threshold</TableCellHead>
+                            <TableCellHead colSpan="2">
+                                Values
+                                <TableCellHead>Expected</TableCellHead>
+                                <TableCellHead>Actual</TableCellHead>
+                            </TableCellHead>
+                            <TableCellHead>Overall Score</TableCellHead>
+                            <TableCellHead colSpan="3">
+                                Regions with divergent score
+                                <TableCellHead>Number</TableCellHead>
+                                <TableCellHead>Percentage</TableCellHead>
+                                <TableCellHead>Name</TableCellHead>
+                            </TableCellHead>
+                        </TableRowHead>
+                    </TableHead>
+                    <TableBody>
+                        {sectionData
```

Comment:
since each subsection is just checking if sectionData exists, it would be better to move that further up